jwt/jwt/jwt.h

```diff
@@ -14,7 +14,7 @@
 #include "zowetypes.h"
 #include "json.h"
 #include "utils.h"
-#include "rs_icsfp11.h"
```

Comment:
What about ```jwt/rscrypto/rs_icsfp11.h```? With the solution you have you may include something by accident (of course with the name rs_icsfp11.h it's unlikely), i.e. the compiler will scan something you didn't intend it to. Plus, you clearly know this is from jwt.